### DISCUSSION TITLE
sign in using phone number

### DISCUSSION THREAD

**Discussion by KhaledEzzet** [serverpod/serverpod#891] [Q&A] [answered]
can I sign in using my phone number?
I use everything in (Setting up Sign in with Firebase) the section in the documentation 
but the problem is FlutterFire UI is deprecated second thing is the documentation to didn't explain how to add a phone auth provider inside  auth providers parameter for example if I use SignInWithFirebaseButton 
what should I put inside authProviders list  to use to phone number auth
SignInWithFirebaseButton(
      ==>          authProviders: [],
                caller: client.modules.auth,
                onFailure: () {},
                onSignedIn: () {},
              ),

**Comment by vlidholt** [comment, marked as answer]
I haven't tried this for Firebase, but it's a feature we are working on natively in Serverpod for the future. We are making some large improvements to authentication in Serverpod 1.2, plus we are updating all the documentation and adding guides.

**Reply by assad369** [reply]
Hey, when you are going to release serverpod 1.2
